src/operators/Remap/QvisRemapWindow.C

```diff
@@ -297,6 +290,30 @@ QvisRemapWindow::UpdateWindow(bool doAll)
                 if(endYLabel)
                     endYLabel->setEnabled(false);
             }
+            if (atts->GetIs3D() == true && atts->GetUseExtents() == false)
+            {
+                startZ->setEnabled(true);
+                if(startZLabel)
```

Comment:
Why this if test for existence of startZLabel (and similar for X,Y)? If the startZ widget exists, shouldn't its label as well?

Reply:
Yes, it should. This code was generated by xml2window, so I'm not entirely sure why it is here. It may act as an extra safeguard against the possibility of a null object in case the code that initializes startZLabel gets modified. Do you want me to go through and remove these?

Reply:
If you look at CreateWindowContents, you can clearly see the labels are created at the same
time as the start and end widgets, so you really shouldn't need to test for their existence.
A lot of code generated by xml2window is designed to be overwritten, it just give you a
starting place. (Not  like xml2atts, xml2cmake, which we expect to be clobberable.)

Reply:
Got it! I've removed these checks